src/kernel_config_manager.sh

```diff
@@ -734,6 +747,10 @@ function parse_kernel_config_manager_options()
         options_values['TARGET']="$REMOTE_TARGET"
         shift 2
         ;;
+      --verbose)
+        options_values['VERBOSE']=1
+        shift
+        ;;
```

Comment:
You forgot to add `verbose` to the `long_options` definition. Without it, the `--verbose` option gets denied by `kw_parse` before getting here.